compiler/rustc_hir/src/attrs/encode_cross_crate.rs

```diff
@@ -70,6 +70,7 @@ impl AttributeKind {
             Naked(..) => No,
             NoCore(..) => No,
             NoImplicitPrelude(..) => No,
+            NoLink => Yes,
```

Comment:
This should be `No`, you can see what it is supposed to be in the `BUILTIN_ATTRIBUTES`